code/modules/mapping/mapping_helpers.dm

```diff
@@ -426,14 +423,17 @@
 	if(!mapload)
 		log_mapping("[src] spawned outside of mapload!")
 		return INITIALIZE_HINT_QDEL
+	return INITIALIZE_HINT_LATELOAD
```

Comment:
I think mapping helpers have a var you are supposed to set for this, `var/late`, rather than returning this manually

Reply:
I saw that quite a few of the mapping helpers with that var enabled still returned `INITIALIZE_HINT_LATELOAD`. If it's redundant then there is no need.

Removed via https://github.com/tgstation/tgstation/pull/86958/commits/9d340c26969d3070c0a70c6a16b4c717a5c3ab55